core/driver/sources/src/main/scala/org/virtuslab/ideprobe/Shell.scala

```diff
@@ -172,5 +171,5 @@ class BaseShell {
     Seq(outputForwarder)
   }
 
-  protected def customizeBuilder(builder: NuProcessBuilder): Unit = ()
```

Comment:
how are inheritors supposed to customizeBuilder if you take away the builder?

Reply:
You're right - it would make no sense. I just did not find any implementation of this method in the source code and wanted to get rid of the warning

Reply:
Huh the other problem might be that... will this builder ever be customized indeed? 😅 @lukaszwawrzyk was the customization added just in case?